compat/elementor/elementor.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+class SiteOrigin_Widgets_Bundle_Elementor {
+
+	/**
+	 * Get the singleton instance
+	 *
+	 * @return SiteOrigin_Widgets_Bundle_Elementor
+	 */
+	public static function single() {
+		static $single;
+
+		return empty( $single ) ? $single = new self() : $single;
+	}
+
+	private $plugin;
+
+	function __construct() {
+		add_action( 'template_redirect', [ $this, 'init' ] );
+
+		add_action( 'wp_ajax_elementor_render_widget', [ $this, 'ajax_render_widget_preview' ] );
+		add_action( 'wp_ajax_elementor_editor_get_wp_widget_form', [ $this, 'ajax_render_widget_form' ] );
+	}
+
+	function init() {
+
+		$this->plugin = Elementor\Plugin::instance();
```

Comment:
No, but we do our own check [here](https://github.com/siteorigin/so-widgets-bundle/blob/3b240d6a98220d1b12decf3c11429900ce7ea9ed/compat/compat.php#L44) which prevents our Elementor compat code from loading.
We could probably just check for `class_exists( 'Elementor\\Plugin' )` because I don't think that class is loaded if PHP version is < 5.4, but I feel safer having the version guard too. :sweat_smile: